src/main/java/com/JSONpostroute/functions/orders/OrderFunctions.java

```diff
@@ -14,4 +14,12 @@ public static int totalOrders(List<RequestJSON> orderList){
         }
         return total_orders;
     }
+
+    public static double totalOrderValue(List<RequestJSON> list){
+        double total_order_value = 0;
+        for (int i = 0; i < list.size(); i++) {
+            total_order_value += (list.get(i).getUnit_price() * list.get(i).getQuantity());
```

Comment:
I'm wondering about the sum of quantity, where you are thinking to get it?

Reply:
Do you mean the sum of all quantities for all orders?

I didn't include that in this function as the price is different for each product, so instead I figure out the total order price for each product (quantity times unit price) then add all the prices together.

Does that make sense? I know I took away the variable names to save memory, but I can add a comment to make it clearer

Reply:
I'm talking about that to save another "for" to get the sum of all quantities. No worries...